khal/khalendar/vdir.py

```diff
@@ -0,0 +1,336 @@
+'''
+Based off https://github.com/pimutils/python-vdir, which is itself based off
+vdirsyncer.
```

Comment:
I don't think there's any work of other copyright holders left in python-vdir. I wrote it as-is, and as I'm already listed in khal's AUTHORS, I think khal's copyright can be used.